.changeset/many-toys-search.md

```diff
@@ -0,0 +1,26 @@
+---
+'sku': minor
+---
+
+Add `libraryFile` configuration option
+
+This option allows the file name of the library to be specified in the sku
+configuration. If this option is not specified then the `libraryName` option
+will be used for this purpose instead (note that this is the previously
+existing behaviour).
```

Comment:
Putting each sentence on its own line here results in better release note formatting IMO.

```suggestion
This option allows the file name of the library to be specified in the sku configuration.
If this option is not specified then the `libraryName` option will be used for this purpose instead (note that this is the previously existing behaviour).
```